### DISCUSSION TITLE
new version of retroarch 1.10.3 and pcsx rearmed r23l aced3eb

### DISCUSSION THREAD

**Discussion by mattitotti** [OnionUI/Onion#646] [General]
hello today i put the onion os 4.1.0 alpha in my new arrived today miyoo mini is awesome, but please can you update the version of retroarch ? in the alpha is the version 1.10.3 and the new version of retroarch is 1.14.0 and the new version of pcsx rearmed is r23l aced3eb, and i can  and i wanna do a contribute to the team, i wanna do a donation for this.
please respond me soon

**Comment by schmurtzm** [comment]
Hi, happy that you join the club today ;)  
I will probably update pcsx core but it's not hurry, the last update only fix a black screen with shadow tower which is not a very played game.

Can I ask why do you want to update retroarch ?  
We pay close attention to Retroarch's commits and it doesn't look like this new version has any advantage for the Miyoo Mini. You know with all Onion users, each change must be tested for a long time before considering to put it in release. So create risks while being sure that it will bring nothing in terms of functionality or performance 🤷‍♂️...

**Reply by mattitotti** [reply]
Only for the update of pcsx rearmed i wanna retroarch update 😄 please i Will really appresceate if you Will do and please can you give me the link for a donation i really wanna thanks you for your work

**Reply by schmurtzm** [reply]
Hi, 
The update of pcsx doesn't require to update Retroarch. I will update it to the last version before releasing Onion 4.1.0 😉.
Here is the link to participate to Onion : https://github.com/sponsors/Aemiii91

**Reply by mattitotti** [reply]
i done my donation, I really wanna thank you for updating it, now i'm in the onion 4.10 alpha, i can't wait for the stable release.

**Comment by mattitotti** [comment]
Please responde me my friend 😄 and thank you for respond me Always
